### ISSUE TITLE
Compatibility with Ti SDK 6.X.X

### ISSUE THREAD

**Issue by dayalav** [progress44/ti.material#1]
When add ti.material to a blank new project with SDK 6.0.1, 6.1.0., 6.1.1 always mark as an invalid module for Titanium
<img width="499" alt="captura de pantalla 2017-07-14 a las 9 49 37" src="https://user-images.githubusercontent.com/4660940/28203388-d9d08af0-6879-11e7-87ae-cf63b75ed393.png">

**Comment by progress44** [comment]
So it shows the same error with 6.0.1 as well? I've tested it with SDK 6.0.4.
It could be because of the android version of the module which isn't implemented yet.

**Comment by dayalav** [comment]
@progress44 Yes, with a new default alloy project. In an old project works well with IOS but not with Android. 
I create a blank project to isolate the error with android, and see the error of this issue.

**Comment by Max87ZA** [comment]
Hi guys,
any news on tracking this bug? I tried to install android version of this module and it says it's not supported for TiSDK 7.2.0.
Thanks

**Comment by cabraljunior** [comment]
Me too

**Comment by icecandy** [comment]
Current version does not work with Ti SDK 7.1.1 (Android).
Tried to rebuild the android module with Ti SDK 7.1.1 and got following error:
[ERROR] Failed to run ndk-build
[ERROR]   
[ERROR] /android/ndk-bundle/build/core/build-local.mk:158: *** Android NDK: Aborting.    .  Stop.
I'm not an expert on modules so not sure what the problem is.

**Comment by cabraljunior** [comment]
Hi, me neither lol
try 
https://stackoverflow.com/questions/41944138/how-to-fix-error158-android-ndk-aborting-stop-ndk-build-cmd-finished-w
